03_Glossary_of_Terms.txt

```diff
@@ -65,11 +72,22 @@ how the NPCs behave and act.
 GM: 
     See Game Master.
 
+Hardpoint:
+    An Armor Hardpoint is a purpose-built universal modular slot that advanced
```

Comment:
Universal and modular are redundant. Also grammar requires separation of adjectives with commas, 
e.x. purpose-built, modular slot